addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -342,6 +342,9 @@ return core.Class.extend(mixins.EventDispatcherMixin, ServicesMixin, {
                     this._log.push("- extra_trigger '" + tip.extra_trigger + "': " + $extra_trigger.length);
                     this._log.push("- visible extra_trigger '" + tip.extra_trigger + "': " + extra_trigger);
                 }
+                if ($trigger.iframeContainer) {
+                    this._log.push("- iframe container: " + $trigger.iframeContainer.contentDocument.body.parentElement.outerHTML);
+                }
```

Comment:
I think we should do it more generically at the body log level: when there is an error, we log the whole body HTML... but we should also get into all iframes and add their content between the `<iframe>` tags automatically

Reply:
Ok